test/decl/protocol/existential_member_access/invalid_type_parameter.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-typecheck-verify-swift -target %target-swift-5.9-abi-triple
+
+// https://github.com/swiftlang/swift/issues/77840
+do {
+  struct G<T> {}
+
+  protocol P {
+    associatedtype A where A == Undefined
```

Comment:
Why is the behavior here any different than the case where `Undefined` exists?

Reply:
The way I think about it is that if reduction returns an error type, then it makes sense to ignore the constraint that caused it to error out.